scripts/mutect2_wdl/mutect2.wdl

```diff
@@ -575,6 +606,21 @@ task M2 {
             ${true='--bam-output bamout.bam' false='' make_bamout} \
             ${"--orientation-bias-artifact-priors " + artifact_prior_table} \
             ${m2_extra_args}
+
+        ### GetPileupSummaries
+        # These must be created, even if they remain empty, as cromwell doesn't support optional output 
+        touch tumor-pileups.table
+        touch normal-pileups.table
+
+        if [[ -f "${variants_for_contamination}" ]]; then
+            gatk --java-options "-Xmx${command_mem}m" GetPileupSummaries -R ${ref_fasta} -I ${tumor_bam} ${"--interval-set-rule INTERSECTION -L " + intervals} \
+                -V ${variants_for_contamination} -L ${variants_for_contamination} -O tumor-pileups.table
+
+            if [[ -f ${normal_bam} ]]; then
```

Comment:
Does this logic work with NIO where the normal bam isn't localized?  I would think `[[ -f ]]` would give false.

Reply:
If this is the case and you have a fix I would do the same for variants for contamination.  It's currently a `File` but we want to future-proof.

Reply:
changed to `! -z`. That should work, will test.